config/locales/en/loan_questions.en.yml

```diff
@@ -37,19 +37,5 @@ en:
         edit_item: "Edit Item"
         delete_item: "Delete Item"
       form:
-        edit_item: "Edit Question or Group"
-        new_item: "New Question or Group"
-
-  simple_form:
```

Comment:
Try it next time and see if it works. I'm not sure if it will. If it does, yes it would be nice.